server/src/pages/shot/text-tool.js

```diff
@@ -240,15 +238,48 @@ exports.TextTool = class TextTool extends React.Component {
   }
 
   adjustX(e) {
-    if (previousInputText === this.textInput.current.textContent) {
+    // if the text hasn't chagned, do nothing
+    if (this.previousInputText === this.textInput.current.textContent) {
+      console.log("no change gtfo");
```

Comment:
nit: We can remove log before merge

Reply:
hah, sorry, that wasn't supposed to be there. :)